src/rocm_docs/__init__.py

```diff
@@ -23,7 +23,7 @@ class ROCmDocs:
         "doxygen_root",
         "doxygen_project",
         "doxyfile",
-        "doxysphinx_enabled",
+        "doxysphinx_enabled"
```

Comment:
This was formatted at my side this way by black, I don't mind any autoformatter but we should agree on one then forget formatting changes.

Reply:
I have no preference for formatting this, so I have added back the comma in [3b79b44](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/117/commits/3b79b44b94f3d14e04143212d438b15de9161642)